Core/Src/msb.c

```diff
@@ -209,4 +182,9 @@ int8_t vcc5_en_write(bool status)
 {
 	HAL_GPIO_WritePin(VCC5_En_GPIO_Port, VCC5_En_Pin, status);
 	return 0;
+}
+
+int32_t imu_data_get(stmdev_ctx_t *ctx, stmdev_ctx_t *aux_ctx, lsm6dso_md_t *imu_md_temp, lsm6dso_data_t *imu_data_temp)
+{
+	lsm6dso_data_get(&ctx, &aux_ctx, &imu_md_temp, &imu_data_temp);
```

Comment:
please acquire the mutex as accel_read and gyro_read did

Reply:
this function should also be in `SENSOR_IMU` blocks.  Annoying ik, but useful for debugging issues.